docs-client/src/lib/transports/thrift.ts

```diff
@@ -58,7 +59,7 @@ export default class ThriftTransport extends Transport {
       hdrs.set(name, value);
     }
 
-    const httpResponse = await fetch(endpoint.pathMapping, {
+    const httpResponse = await fetch(endpointPath || endpoint.pathMapping, {
```

Comment:
Realize that I need to use endpointPath parameter to fetch correct endpoint and thriftMethod.